Makefile

```diff
@@ -605,11 +605,11 @@ else
 LLVM_SIZE := $(build_depsbindir)/llvm-size$(EXE)
 endif
 build-stats:
-	@echo $(JULCOLOR)' ==> ./julia binary sizes'$(ENDCOLOR)
```

Comment:
I believe we usually use `printf` for formatted printing, because the POSIX compliant behavior for `echo -e` is to print the `-e`